### ISSUE TITLE
[Rule] *** Settings *** should be left aligned

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#462]
I'm not sure if we have anything that covers that but this code:
```
*** Settings ***
   Library  Collections
```
produces
```
0401 Robot Framework syntax error: Non-existing setting ''. (parsing-error)
```
but it should report it as not aligned statement.
